scripts/install_build_tools.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eu
+
+unset -v progdir
+case "${0}" in
+/*) progdir="/";;
+*/*) progdir="${0%/*}";;
+*) progdir=".";
+esac
```

Comment:
if the current directory path follows the pattern  `/*` then `progdir` will be `/` which gives the wrong path in line 12, because line 12 assumes the `/` won't be there and it hard-coded the slash. 
from line 5 to line 10 could be replaced with this simple command:

```shell
progdir="$(dirname "${0}")"
```

Reply:
Hey, I just restored the file that was deleted. Feel free to make edits in a separate PR. Now that the build is working it'll be pretty easy to test. Thanks!

Reply:
great